canonical-web-and-design/ubuntu.com
# UA shop form changes

## Done

Removed address textarea from UA payment modal and replaced it with specific inputs for address components, along with a new "Account Name" field.

We additionally show a list of states/provinces respectively if the user sets US/Canada as their country.

## QA

- Check out this branch
- `dotrun` it
- Visit /advantage/subscribe
- Select a product, add it to your cart and click "Buy now"
- See the additional address fields, and fill them out
- Card details: 4242 4242 4242 4242, 04/24, 242, 24242
- Set your country to US, see that a states select appears
- Set your country to Canada, see that a provinces select appears
- Set your country to something other than US or Canada, see that no State or Province select is visible
- When the form is complete and valid, click "Continue"
- Accept ts&cs and click "Pay"
- See that there are no errors, and that you see a success notification

Fixes #8738
